simpeg_drivers/joint/joint_surveys/driver.py

```diff
@@ -16,17 +16,14 @@
 # ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
 
-# pylint: disable=unexpected-keyword-arg, no-value-for-parameter
-
 from __future__ import annotations
 
 import numpy as np
 from geoh5py.shared.utils import fetch_active_workspace
 from SimPEG import maps
 
-from simpeg_drivers.components.factories import DirectivesFactory
```

Comment:
not sure if relative import is mandatory here for pylint to work, but `__init__.py` probably was

Reply:
Yea I figured it out. Missing an init__ in there...